packages/pkgs-by-name/ghaf-vms/package.nix

```
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: 2022-2026 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  writeShellApplication,
+  lib,
+  busybox,
+  systemd,
+}:
+let
+  colors = {
+    normal = "\\033[0m";
+    red = "\\033[0;31m";
+    green = "\\033[0;32m";
+    yellow = "\\033[0;33m";
+  };
```

Missing bold red for "FAILED" status of VM.


```suggestion
  colors = {
    normal = "\\033[0m";
    red = "\\033[0;31m";
    boldRed = "\\033[1;31m";
    green = "\\033[0;32m";
    yellow = "\\033[0;33m";
  };
```